test/integrate/dubbo/go-client/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang
```

Comment:
is licence necessary?

Reply:
Okay, I have modified it in the latest commit.